examples/tv-casting-app/android/App/app/src/main/java/com/chip/casting/app/CertTestFragment.java

```diff
@@ -262,6 +304,45 @@ private void runCertTests(Activity activity) {
               kContentApp, successFailureCallbackInteger, successFailureCallbackInteger);
         });
 
+    runAndWait(
+        "mediaPlayback_subscribeToCurrentState",
+        successFailureCallback,
+        () -> {
+          tvCastingApp.mediaPlayback_subscribeToCurrentState(
+              kContentApp,
+              new SuccessCallback<MediaPlaybackTypes.PlaybackStateEnum>() {
+                @Override
+                public void handle(MediaPlaybackTypes.PlaybackStateEnum response) {
+                  // Lets wait for the timeout to avoid the race condition issue in the SDK with
+                  // ReadClient::Close()
+                  // successFailureCallback.handle(MatterError.NO_ERROR);
+                  addCertTestStatus(
+                      activity, MatterError.NO_ERROR, "mediaPlayback_subscribeToCurrentState");
+                }
+              },
+              successFailureCallback,
+              0,
```

Comment:
add as comments what the hard coded numbers mean

Reply:
Ack. Will add in the next update.